vulture/utils.py

```diff
@@ -37,6 +38,17 @@ def read_file(filename):
         raise VultureInputException(err)
 
 
+def return_nodes_after_return(tree):
+    try:
+        for i, node in enumerate(tree.body, 0):
+            if isinstance(node, ast.Return):
+                return tree.body[i+1:]
+    except AttributeError:
+        return []
+    else:
+        return return_nodes_after_return(node)
+
```

Comment:
I think it's best to incorporate this functionality into the current design by adding this method at the end of the Vulture methods

```python
    def generic_visit(self, node):
        """Called if no explicit visitor function exists for a node."""
        for field, value in ast.iter_fields(node):
            if isinstance(value, list):
                self._handle_ast_list(value)
                for item in value:
                    if isinstance(item, ast.AST):
                        self.visit(item)
            elif isinstance(value, ast.AST):
                self.visit(value)
```
and this method above the `visit()` method:

```python
    def _handle_ast_list(self, ast_list):
        for index, node in enumerate(ast_list):
            if isinstance(node, ast.Return):
                try:
                    first_unreachable_node = ast_list[index + 1]
                except IndexError:
                    pass
                else:
                    self._define(
                        self.unreachable_code,
                        'code after return',
                        first_unreachable_node.lineno,
                        size=(
                            lines._get_last_line_number(ast_list[-1]) -
                            first_unreachable_node.lineno + 1))
```

The most important part, however, is creating test cases. Remember that return statements can occur in very many places in Python.